score_detection.py

```diff
@@ -21,52 +7,84 @@
 use_stream = False
 
 def cmdsf(list, loc):
-    global frames_to_skip
-
-    if loc + 1 < len(list):
-        frames_to_skip = int(list[loc + 1])
+	global frames_to_skip
+	if loc+1 < len(list):
+		frames_to_skip = int(list[loc+1])
 
 def cmdhelp(list,loc):
-    print "Usage: python score_detection.py <options>"
-    print "Options:"
-    print "--skipframes <number>: Sets the amount of frames skipped between reads"
-    print "--help: Shows usage and options, then exits"
-    print "--use_stream: Use twitch stream instead of file"
-    print "--read_from <filename>: Reads from a custom filename. Reads custom url if you are scanning a Twitch stream"
-
-    exit()
-
+	print "Usage: python score_detection.py <options>"
+	
+	print "Options:"
+	
+	print "--skipframes <number> : Sets the amount of frames skipped between reads"
+	
+	print "--help : Shows usage and options, then exits"
+	
+	print "--use_stream : Use twitch stream instead of file"
+	
+	print "--read_from <filename> : Reads from a custom filename. Reads custom url if you are scanning a Twitch stream"
+	
+	exit()
+	
 def cmdreadfrom(list,loc):
-    global read_link
 
-    if loc + 1 < len(list):
-        read_link = str(list[loc+1])
+	global read_link
+	
+	if loc+1 < len(list):
+	
+		read_link = str(list[loc+1])
 
 def cmdusestream(list,loc):
-    global use_stream
 
-    use_stream = True
+	global use_stream
+	
+	use_stream = True
 
 cmdoptions = {
-    "--skipframes": cmdsf,
-    "--help" : cmdhelp,
-    "--use_stream" : cmdusestream,
-    "--read_from" : cmdreadfrom
+
+ "--skipframes": cmdsf,
+ 
+ "--help" : cmdhelp,
+ 
+ "--use_stream" : cmdusestream,
+ 
+ "--read_from" : cmdreadfrom
 }
 
 def parseCommandArgs():
-    cmdlist = sys.argv[1:len(sys.argv)]
 
-    if len(cmdlist) == 0:
-        print "Usage: python score_detection.py <options>"
-        print "Use the --help option to get a list of possible options."
+	cmdlist = sys.argv[1:len(sys.argv)]
+	
+	if len(cmdlist) == 0:
+	
+		print "Usage: python score_detection.py <options>"
+		
+		print "Use the --help option to get a list of possible options."
+		
+		exit()
+		
+	for i in range(0,len(cmdlist)):
+	
+		if str(cmdlist[i]) in cmdoptions.keys():
+		
+			cmdoptions[cmdlist[i]](cmdlist,i)
 
-        exit()
+parseCommandArgs()
 
-    for i in range(0,len(cmdlist)):
-        if str(cmdlist[i]) in cmdoptions.keys():
-            cmdoptions[cmdlist[i]](cmdlist,i)
+import cv2
+import numpy as np
+import pytesseract
+import re
+from PIL import Image
```

Comment:
I'll implement a module that handles command line args. And yes, I did test it. Significant performance boost. Before, if I typed ```python score_detection.py --help```, it would take 5 to 10 seconds to import the libraries (depended on the computer being used). Now, even on my Raspberry Pi, it executes in less than half a second.